notebook/flaml_forecast.ipynb

```diff
@@ -63,14 +65,14 @@
    "cell_type": "code",
    "execution_count": 2,
    "source": [
-    "# split the data into a train dataframe and X_test and y_test dataframes, where the number of samples for test is equal to\n",
-    "# the number of periods the user wants to predict\n",
-    "num_samples = data.shape[0]\n",
-    "time_horizon = 12\n",
-    "split_idx = num_samples - time_horizon\n",
-    "X_train = data[:split_idx]  # X_train is a dataframe with two columns: time and value\n",
-    "X_test = data[split_idx:]['index'].to_frame()  # X_test is a dataframe with dates for prediction\n",
-    "y_test = data[split_idx:]['co2']  # y_test is a series of the values corresponding to the dates for prediction"
+    "# split the data into a train dataframe and X_test and y_test dataframes, where the number of samples for test is equal to\r\n",
+    "# the number of periods the user wants to predict\r\n",
+    "num_samples = data.shape[0]\r\n",
+    "time_horizon = 12\r\n",
+    "split_idx = num_samples - time_horizon\r\n",
+    "X_train = data[:split_idx] # X_train is a dataframe with two columns: time and value\r\n",
+    "X_test = data[split_idx:]['index'].to_frame() # X_test is a dataframe with dates for prediction\r\n",
+    "y_test = data[split_idx:]['co2'].to_frame() # y_test is a dataframe of the values corresponding to the dates for prediction"
```

Comment:
Recover

Reply:
Do you mean recover the end of line? or the other changes or both?